v2-protocol.md

```diff
@@ -37,6 +37,16 @@
 | WithdrawalManager | [`0x7ED195a0AE212D265511b0978Af577F59876C9BB`](https://etherscan.io/address/0x7ED195a0AE212D265511b0978Af577F59876C9BB) |
 | PoolDelegateCover | [`0x9c74C5147653041239bb31C799c54767D9953f7D`](https://etherscan.io/address/0x9c74C5147653041239bb31C799c54767D9953f7D) |
 
+## Maven11 USDC-3 Pool Contracts
+
+| Contract | Address |
+| -------- | ------- |
+| Pool              | [`0xd2b01f8327eeca47829efc731f1a89c6d07e6b92`](https://etherscan.io/address/0xd2b01f8327eeca47829efc731f1a89c6d07e6b92) |
+| PoolManager       | [`0x83376e49c7bf776495313c89f9bf226c8b8971b6`](https://etherscan.io/address/0x83376e49c7bf776495313c89f9bf226c8b8971b6) |
+| LoanManager       | [`0x9b300a28d7dc7d422c7d1b9442db0b51a6346e00`](https://etherscan.io/address/0x9b300a28d7dc7d422c7d1b9442db0b51a6346e00) |
```

Comment:
Should we differentiate between fixed and open term loan managers from now on?

Reply:
Yh going forward we should but suggest waiting for the upgrade to do so